src/components/UI/InputImage/InputImage.component.js

```diff
@@ -34,13 +35,9 @@ class InputImage extends PureComponent {
   blobToBase64(blob) {
     const reader = new FileReader();
 
-    reader.addEventListener(
-      'load',
-      () => {
-        this.props.onChange(reader.result);
-      },
-      false
-    );
+    reader.onload = () => {
+      this.props.onChange(reader.result);
+    };
```

Comment:
@RodriSanchez1 Did you test this both for android and electron?

Reply:
Yes, i did it in both platforms!